example/android/build.gradle

```diff
@@ -6,14 +6,14 @@ buildscript {
         minSdkVersion = 16
         compileSdkVersion = 28
         targetSdkVersion = 28
-        supportLibVersion = "28.0.0"
+        supportLibVersion = "1.0.0"
```

Comment:
This can be left as the default.

Reply:
It can. I wrote jetifier, I'm pretty aware of this stuff, but you're right. Not in the template.